README.md

```diff
@@ -38,7 +38,23 @@ or start on the [landing page](http://localhost:8000/)
 6. Todos should be stored in MySQL Database, with a persistent volume connected
 7. Create PR with your changes and attach it for validation on a platform
 
-
+8. Download the application from github 
```

Comment:
Separate your part from the existing doc.

Reply:
fixed